dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -158,6 +162,37 @@ class ScriptsControllerTest < ActionController::TestCase
     end
   end
 
+  test "show: redirect to latest stable version in family for logged out user" do
+    get :show, params: {id: @coursez_2017.name}
+    assert_redirected_to "/s/#{@coursez_2018.name}?redirect_warning=true"
+  end
+
+  test "show: do not redirect to latest stable version if no_redirect query param is supplied" do
+    get :show, params: {id: @coursez_2017.name, no_redirect: "true"}
```

Comment:
similar to above, it would make this test case more clear to add a get / assert above this line showing the behavior without no_redirect